plugins/test/unit/plugins/catalogers/test_rhui.py

```diff
@@ -25,7 +25,7 @@ def test_metadata(self):
         expected = {
             'id': TYPE_ID,
             'display_name': "RHUI Cataloger",
-            'types': [models.RPM.TYPE]
+            'types': [ids.TYPE_ID_RPM]
```

Comment:
Better to use `models.RPM._content_type_id` instead of constant in `ids` module?